packages/plugins/rubygem-hammer_cli_foreman_kubevirt/rubygem-hammer_cli_foreman_kubevirt.spec

```diff
@@ -0,0 +1,94 @@
+# Generated from hammer_cli_foreman_kubevirt-0.1.0.gem by gem2rpm -*- rpm-spec -*-
+# template: hammer_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name hammer_cli_foreman_kubevirt
+%global plugin_name foreman_kubevirt
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+%global hammer_confdir %{_root_sysconfdir}/hammer
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.0
+Release: 1%{?foremandist}%{?dist}
+Summary: Foreman kubevirt commands for Hammer CLI
+Group: Development/Languages
+License: GPL-3.0
+URL: https://github.com/theforeman/hammer-cli-foreman-kubevirt
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(gettext) >= 3.1.3
+Requires: %{?scl_prefix}rubygem(gettext) < 4.0.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) <= 0.17.0
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+Foreman kubevirt commands for Hammer CLI.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+mkdir -p %{buildroot}%{hammer_confdir}/cli.modules.d
+install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
+                %{buildroot}%{hammer_confdir}/cli.modules.d/%{plugin_name}.yml
```

Comment:
The build fails on this line:
```
+ install -m 0644 ./opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli_foreman_kubevirt-0.1.0/config/foreman_kubevirt.yml /builddir/build/BUILDROOT/tfm-rubygem-hammer_cli_foreman_kubevirt-0.1.0-1.fm1_23.el7.noarch/etc/hammer/cli.modules.d/foreman_kubevirt.yml
BUILDSTDERR: install: cannot stat './opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli_foreman_kubevirt-0.1.0/config/foreman_kubevirt.yml': No such file or directory
```

This is probably because you're not including the config in the gem. https://github.com/theforeman/hammer-cli-foreman-kubevirt/blob/6770db5f952eac4986ae42946f32ac7d65775603/hammer_cli_foreman_kubevirt.gemspec#L13 should also list it. See https://github.com/theforeman/hammer-cli-foreman/blob/1789509b1d9991cab82e43aa281f734756c5d604/hammer_cli_foreman.gemspec#L21 as well.